src/yang/ietf-qos-action.yang

```diff
@@ -541,22 +544,17 @@ module ietf-qos-action {
 
       leaf burst-value {
         type uint64;
+        units "bytes";
```

Comment:
since this is for Queues, let us retain the units. For Meters, it is fine since there is RFC reference.